enterprise/server/remote_execution/runner/runner.go

```diff
@@ -89,9 +89,9 @@ const (
 	removed
 
 	// How long to spend waiting for a runner to be removed before giving up.
-	runnerCleanupTimeout = 30 * time.Second
+	runnerCleanupTimeout = 3 * time.Minute
```

Comment:
I think this is unrelated to the pause timeouts so we might want to keep as-is unless we find that the current timeout is too short